app/helpers/application_helper/button/policy_edit.rb

```diff
@@ -0,0 +1,6 @@
+class ApplicationHelper::Button::PolicyEdit < ApplicationHelper::Button::PolicyButton
+
+  def role_allows_feature?
+    role_allows?(:feature => "policy_edit")
```

Comment:
Its needed, because I use this class for multiple buttons which have different id, but should be checked as `policy_edit`